src/main/resources/locales/cy/common.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
This feels like it should be a custom nunjucks filter (look in src/modules/nunjucks/index.ts for some examples). This way you could do something like this as a filter

env.addFilter('month', function(x, y){x == 'en' ? y : monthLookup[y];});

and then to call in frontend, it'd be like 

{{ month | month(lng) }}